staging/src/k8s.io/kubectl/pkg/cmd/create/create_token.go

```diff
@@ -103,6 +104,13 @@ var (
 	}
 )
 
+func init() {
```

Comment:
slight preference for making boundObjectKindToAPIVersion a getter so we can unit test if we want

```go
func boundObjectKindToAPIVersion(kind string) (version string, ok bool) {
  switch {
  case kind == "Pod":
    return "v1", true
  case kind == "Secret":
    return "v1", true
  case kind == "Node" && os.Getenv("KUBECTL_NODE_BOUND_TOKENS") == "true":
    return "v1", true
  default:
    return "", false
  }
}
```

Reply:
done (and fixed tests)